pandas/_testing/asserters.py

```diff
@@ -1158,7 +1158,10 @@ def assert_frame_equal(
         Specify how to compare internal data. If False, compare by columns.
         If True, compare by blocks.
     check_exact : bool, default False
-        Whether to compare number exactly.
+        Whether to compare number exactly. If False, the comparison uses the
+        relative tolerance (`rtol`) and absolute tolerance (`atol`)
+        parameters to determine if two values are considered close,
+        according to the formula: `|a - b| <= (atol + rtol * |b|)`.
```

Comment:
```suggestion
        relative tolerance (``rtol``) and absolute tolerance (``atol``)
        parameters to determine if two values are considered close,
        according to the formula: ``|a - b| <= (atol + rtol * |b|)``.
```

Reply:
Thanks Matthew, I've changed the docs following your code suggestion.